DSCResources/xUAC/xUAC.schema.psm1

```diff
@@ -20,7 +20,13 @@ Configuration xUac
             $ConsentPromptBehaviorAdmin = 2
             $EnableLua = 1
             $PromptOnSecureDesktop = 1
-        }    
+        }
+        "AlwaysNotifyAndAskForCredentials" 
+        {
```

Comment:
I think this should be 1 rather than 3 since this uses the secure desktop where 3 does not.

https://msdn.microsoft.com/en-us/library/cc232761.aspx